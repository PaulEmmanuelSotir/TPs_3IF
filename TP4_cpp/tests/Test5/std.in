EN_ERROR_MESS
PC poc1 0 0 0 10 10 8 8 4
PC poc2 0 0 0 10 10 8 5 5
PC poc3 0 0 0 1 1 1 1 0
PC poc4 0 0 0 1
EXIT
