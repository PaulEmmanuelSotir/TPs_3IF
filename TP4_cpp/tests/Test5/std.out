OK
OK
Error: Invalid polygon
OK
Error: Wrong number of arguments
